static/src/javascripts/test/spec/common/commercial/third-party-tags/outbrain.spec.js

```diff
@@ -125,6 +125,15 @@ describe('Outbrain', function () {
             mediator.emit('modules:commercial:dfp:rendered', eventStub);
             expect(sut.load).not.toHaveBeenCalled();
         });
+
+        it('should load when ad block is in use', () => {
```

Comment:
Maybe we should add an explanation why we do this? Just for whoever looks at it next.